.release-please-manifest-submodules.json

```diff
@@ -0,0 +1,111 @@
+{
+  "accessapproval": "1.0.1",
```

Comment:
Will this need to be manually updated when a new submodule is added? Or is this something that is auto-updated?

Reply:
Yes, this will need to be updated for new modules.